### ISSUE TITLE
Levers "jumping" when drag and move

### ISSUE THREAD

**Issue by deiks** [flybywiresim/aircraft#8448]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-01-28T15:02:11+00:00",
    "ref": "refs/heads/master",
    "sha": "41f8f0b7acc1ad2483e67bd7b7f851a0ac1c36c4",
    "actor": "Gurgel100",
    "event_name": "manual",
    "pretty_release_name": "master:41f8f0b7",
    "version": "v0.12.0-dev.41f8f0b"
}
```

### Describe the bug

When moving speed brake lever, lever "jumps".

https://github.com/flybywiresim/aircraft/assets/4221827/5ba63c3a-92a7-40f6-9f39-745689011746

According to _Mav [z+1 de en sv]_, the same issue happens for flaps lever as well, but less noticable.

First time noticed when new cockpit redesign version is released few weeks ago.

### Expected behavior

Expecting to move it normally without "jumping" :D

### Steps to reproduce

1. Load sim
2. Try to engage into armed position from retracted position

### Discord Username (optional)

DX#2539

**Comment by frankkopp** [comment]
I can confirm this - see also my video

https://github.com/flybywiresim/aircraft/assets/16833201/041db188-a83f-41bd-be31-c4f2302a7a0f

**Comment by KOWA380** [comment]
+1 can confirm

**Comment by Maximilian-Reuter** [comment]
Yes I tested and this only happens in the new cockpit so I assume it has smth to do with that. 
@tracernz are you aware of this ?

**Comment by tracernz** [comment]
Yeah, as discussed, there are some higher priority items on my radar for the moment.
